Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM debian:jessie
```

Comment:
Nope. I created the ``Dockerfile`` to be able to quickly spin up a developer environment. I have a mac, therefore I cannot run the app locally. Docker allowed me to quickly solve that.

The ``Vagrantfile`` is more to test the ``Ansible`` playbook, although  it could also be used for developing. However I think it is way less practical for that matter.